samples/sampler/stories/qa/Spotlight.js

```diff
@@ -518,6 +518,35 @@ export const FocusRing = () => (
 
 FocusRing.storyName = 'Spottable with Focus Ring';
 
+export const FocusWithPreventScroll = () => {
+	const handleClick1 = useCallback(() => {
+		Spotlight.focus('#FocusWithPreventScrollTarget1');
+	}, []);
+
+	const handleClick2 = useCallback(() => {
+		Spotlight.focus('#FocusWithPreventScrollTarget2', {preventScroll: true});
+	}, []);
+
+	return (
+		<>
+			<p>Click on the first button calls Spotlight.focus() to focus the second button for each scroller.</p>
```

Comment:
I guess it's not working when you "click" by pointer.
So "Click" word makes confusing. Let's reword this to Press 5way OK or enter key?

Reply:
Sure, it looks much better!